unet/unet.cpp

```diff
@@ -85,37 +86,62 @@ ILayer* up(INetworkDefinition *network, std::map<std::string, Weights>& weightMa
     for (int i = 0; i < resize * 2 * 2; i++) {
         deval[i] = 1.0;
     }
-    Weights emptywts{DataType::kFLOAT, nullptr, 0};
-    Weights deconvwts1{DataType::kFLOAT, deval, resize * 2 * 2};
-    IDeconvolutionLayer* deconv1 = network->addDeconvolutionNd(input1, resize, DimsHW{2, 2}, deconvwts1, emptywts);
-    deconv1->setStrideNd(DimsHW{2, 2});
-    deconv1->setNbGroups(resize);
-    weightMap["deconvwts."+lname] = deconvwts1;
-
-    int diffx = input2.getDimensions().d[1]-deconv1->getOutput(0)->getDimensions().d[1];
-    int diffy = input2.getDimensions().d[2]-deconv1->getOutput(0)->getDimensions().d[2];
-    // IPoolingLayer* pool1 = network->addPooling(dcov1, PoolingType::kMAX, DimsHW{2, 2});
-    // pool1->setStrideNd(DimsHW{2, 2});
-    // dcov1->add_pading
-    ILayer* pad1 = network->addPaddingNd(*deconv1->getOutput(0),DimsHW{diffx / 2, diffy / 2},DimsHW{diffx - (diffx / 2), diffy - (diffy / 2)});
-    // dcov1->setPaddingNd(DimsHW{diffx / 2, diffx - diffx / 2},DimsHW{diffy / 2, diffy - diffy / 2});
-    ITensor* inputTensors[] = {&input2,pad1->getOutput(0)};
-    auto cat = network->addConcatenation(inputTensors, 2);
-    assert(cat);
-    if (midch==64){
-        ILayer* dcov1 = doubleConv(network,weightMap,*cat->getOutput(0),outch,3,lname+".conv",outch);
-        assert(dcov1);
-        return dcov1;
+
+    if (BILINEAR){
+        // add upsample bilinear
```

Comment:
can we check the coding style for if-else or for, pay attention to the space, like 

```
if () {
} else {
}

for () {
}
```

Reply:
can you the entire unet.cpp for coding style?